### DISCUSSION TITLE
Idea: Wikidata identifiers as topics

### DISCUSSION THREAD

**Discussion by nathangathright** [Podcastindex-org/podcast-namespace#544] [Ideas]
Inspired by @misener’s article about Apple tagging individual podcast episodes with topics. I was curious if we should explore the merit of creating a `<podcast:topics>` namespace tag based on [Wikidata identifiers](https://www.wikidata.org/wiki/Wikidata:Identifiers#Wikidata_identifiers). From the [article](https://wearebumper.com/blog/2022/11/15/apple-episode-topics):

>  Not only could this lead to new and better podcast discovery features within Apple Podcasts, greater use of structured data also represents a potential win for podcast search and audio SEO.
>
> …It’s not difficult to see how Apple could use relevance-weighted topic data to significantly improve podcast episode discovery…
>
> For example, imagine being able to explore episodes on a particular topic, published in the last week/month/year…Or imagine diving into the back catalog of a long-running show through the lens of topics you care most about.

Obviously, I’d expect to see a lot of keyword-stuffing from "growth hacker" types, but that was also a concern with the person tag. Perhaps there’s a way to mitigate the incentives to act in bad faith.

**Comment by jamescridland** [comment]
If helpful to know, https://podnews.net/article/apple-topics is a viewer for those topics.

The person tag isn't used by larger platforms, partially because of the possibility of spam. But I think it's something that can be mitigated, I agree.

**Comment by thebells1111** [comment]
This would also solve the issue of categorizing music for enhanced searching.

**Comment by ryanwi** [comment]
@nathangathright @jamescridland checking this out from the [Mastodon thread](https://hachyderm.io/@nathan@xoxo.zone/116856488453545499).  Did Apple stop doing this? I took a look at the article and Apple pages today, and I don't see anything today for the topics?

Either way, thanks for pointing out the Wikidata identifiers, this is super helpful and I'm going to adopt Wikidata identifiers where they are available.

**Reply by nathangathright** [reply]
It was only ever something hidden within the raw HTML of Apple’s webpage. They potentially still do this kind of classification internally, but it’s no longer visible to us.
